app/controllers/users_controller.rb

```diff
@@ -229,15 +233,30 @@ def change_email(user_data)
 
   def upload_export_files(user_data)
     logger.info "Start importing account"
-    @user.export = user_data[:export] if user_data[:export]
-    @user.exported_photos_file = user_data[:exported_photos_file] if user_data[:exported_photos_file]
-    if @user.save
-      flash.now[:notice] = "Your account migration has been scheduled"
+
+    import_parameters = copy_import_files(user_data)
+
+    if has_import_parameter?(import_parameters)
+      flash.now[:notice] = t("users.import.import_has_been_scheduled")
     else
-      flash.now[:error] = "Your account migration could not be scheduled for the following reason:"\
-                          " #{@user.errors.full_messages}"
+      flash.now[:error] = t("users.import.import_has_no_files_received")
+    end
+    Workers::ImportUser.perform_async(@user.id, import_parameters)
+  end
+
+  def copy_import_files(user_data)
+    {
+            profile_path: copy_import_file(user_data[:export]),
+            photos_path: copy_import_file(user_data[:exported_photos_file])
+    }
+  end
+
+  def copy_import_file(tmp_file)
+    if tmp_file.present?
```

Comment:
Style/GuardClause: Use a guard clause (`return unless tmp_file.present?`) instead of wrapping the code inside a conditional expression.